File: prime/prime.py
from sys import argv
from math import sqrt


def check_prime(n):
	prime = True
	try:
		n = int(n)
		if n > 1:
			for i in range(2, int(sqrt(n)+1)):
				if n % i == 0:
					prime = False
			if prime == True:
				print "yes"
			else:
				print "no"
		else:
			print "no"
	except ValueError:
		print "warning: \'%s\' is not an integer" %n


checkPrime(argv[1])

File: prime/prime.js
var checkPrime = function(input) {
	var n = parseInt(input, 10);
	if(n > 1){
		var prime = true;
		for(i=2; i<(Math.round(Math.sqrt(n)))+1; i++){
			if(n % i ===0){
				prime = false;
			}
		}
		if(prime === true){
			console.log("true");
		}
		else{
			console.log("false");
		}
	}
	else {
		console.log("false");
	}
};


checkPrime(process.argv[2]);
